NEWS.md

```diff
@@ -1,6 +1,8 @@
 
 # vctrs (development version)
 
+* New `vec_is_list()` for detecting if a vector is a list in the vctrs sense.
```

Comment:
For instance, objects of class `lm` are not lists. In general classes need to explicitly inherit from `"list"` to be considered as lists by vctrs.